src/test/groovy/org/c02e/jpgpj/EncryptorSpec.groovy

```diff
@@ -1,16 +1,24 @@
 package org.c02e.jpgpj
 
+import org.bouncycastle.jce.provider.BouncyCastleProvider
 import org.c02e.jpgpj.key.KeyForDecryption
 import org.c02e.jpgpj.key.KeyForEncryption
 import org.c02e.jpgpj.key.KeyForSigning
 import org.c02e.jpgpj.key.KeyForVerification
 import org.bouncycastle.openpgp.PGPException
+import spock.lang.Ignore
 import spock.lang.Specification
 
+import java.security.Security
+
 class EncryptorSpec extends Specification {
     def cipherOut = new ByteArrayOutputStream()
     def plainOut = new ByteArrayOutputStream()
 
+    def setupSpec() {
+        Security.addProvider(new BouncyCastleProvider())
```

Comment:
We won't need this (here, and same for the other test specs) if we change the `JcaProviderHelper` to automatically use the `BouncyCastleProvider` when available.